content/posts/gitops/index.md

```diff
@@ -0,0 +1,497 @@
+---
```

Comment:
I think you forgot to compile (by running `hugo`), no?

Reply:
Right. Done.